### PR TITLE
[Fluid] Remove fluid materials and flat map warnings

### PR DESCRIPTION
**Pull request by rubenzorrilla**
This PR removes all the warnings related to the material reading and the model flat map retrieve.

In a future PR I will remove the ones related to the `Initialize` method of elements and conditions.

### CODE REVIEW

File: applications/FluidDynamicsApplication/python_scripts/navier_stokes_embedded_solver.py [hunk 1]
```diff
@@ -539,7 +539,8 @@ def __CreateDistanceModificationProcess(self):
         # Note that the distance modification process is applied to the volume model part
         distance_modification_settings = self.settings["distance_modification_settings"]
         distance_modification_settings.ValidateAndAssignDefaults(self.__GetDistanceModificationDefaultSettings(self.level_set_type))
-        distance_modification_settings["model_part_name"].SetString(self.settings["volume_model_part_name"].GetString())
+        aux_full_volume_part_name = self.settings["model_part_name"].GetString() + "." + self.settings["volume_model_part_name"].GetString()
+        distance_modification_settings["model_part_name"].SetString(aux_full_volume_part_name)
```
Comment (philbucher) [inline applications/FluidDynamicsApplication/python_scripts/navier_stokes_embedded_solver.py:543]: why do you manually do this here?
wouldn't it be cleaner to do it in json like or the processes

File: applications/FluidDynamicsApplication/tests/TwoFluidInletTest/materials.json [hunk 2]
```diff
@@ -20,7 +20,7 @@
                 "name" : "Newtonian2DLaw"
             },
             "Variables"        : {
-                "DENSITY": 1.0E+03,
+                "DENSITY": 1.0E+00,
```
Comment (philbucher) [inline applications/FluidDynamicsApplication/tests/TwoFluidInletTest/materials.json:23]: no changes in results?

**Review by philbucher — APPROVED** [review]
minor comment otherwise thanks

File: applications/FluidDynamicsApplication/python_scripts/navier_stokes_embedded_solver.py (hunk 1 above)
Comment (rubenzorrilla) [reply applications/FluidDynamicsApplication/python_scripts/navier_stokes_embedded_solver.py:543]: The distance correction must be always applied to the volume model part. I think it's better to hardcode it to ensure this is accomplished.

File: applications/FluidDynamicsApplication/tests/TwoFluidInletTest/materials.json (hunk 2 above)
Comment (rubenzorrilla) [reply applications/FluidDynamicsApplication/tests/TwoFluidInletTest/materials.json:23]: Only the level set is checked in this test.

**Comment by philbucher** [comment]
ok cool then please go ahead :+

**Comment by rubenzorrilla** [comment]
Merging to avoid blocking #6024.

**Pull request merged**